sklearn/tests/test_naive_bayes.py

```diff
@@ -270,12 +273,11 @@ def test_discretenb_partial_fit(cls):
         assert_array_equal(clf1.feature_count_, clf3.feature_count_)
 
 
-@pytest.mark.parametrize('cls', [BernoulliNB, MultinomialNB, GaussianNB,
-                                 CategoricalNB])
-def test_discretenb_pickle(cls):
-    # Test picklability of discrete naive Bayes classifiers
+@pytest.mark.parametrize('NaiveBayes', ALL_NAIVE_BAYES_CLASSES)
+def test_naive_bayes_pickle(NaiveBayes):
+    # Test picklability of naive Bayes classifiers
```

Comment:
I think we can remove this test as pickability is tested in `check_estimators_pickle` (see utils/estimator_checks.py) in `tests/test_common.py`.
I verified it by running `pytest sklearn/tests/test_common.py -v -k BernoulliNB` and so on.

Reply:
Sounds good. Done in a43f8e520.